R/rdeephaven/R/update_by_ops_wrapper.R

```diff
@@ -111,18 +623,78 @@ uby_ema_tick <- function(decay_ticks, cols = character(), operation_control = op
   return(UpdateByOp$new(INTERNAL_ema_tick(decay_ticks, cols, operation_control$.internal_rcpp_object)))
 }
 
+#' @name
+#' uby_ema_time
+#' @title
+#' Exponential moving average by group with time as the decay unit
+#' @md
+#'
 #' @description
 #' Creates an exponential moving average (EMA) UpdateByOp for each column in `cols`, using time as the decay unit.
+#'
+#' @details
 #' The formula used is
-#'   `dt_current = current_timestamp - prev_timestamp`
-#'   `a_current = e^(-dt_current / decay_time)`
-#'   `ema_first = first_value`
-#'   `ema_current = a_current * ema_prev + (1 - a_current) * current_value`
+#' \deqn{a_i = e^{\frac{-dt_i}{\tau}}}
+#' \deqn{\bar{x}_0 = x_0}
+#' \deqn{\bar{x}_i = a_i*\bar{x}_{i-1} + (1-a_i)*x_i}
+#'
+#' Where:
+#' - \eqn{dt_i} is the difference between time \eqn{t_i} and \eqn{t_{i-1}} in nanoseconds.
+#' - \eqn{\tau} is `decay_time` in nanoseconds, an input parameter to the method.
```

Comment:
I find "in nanoseconds" to be confusing here since users are specifying the duration string, which could be specified in hours or maybe days.